lib/oli_web/templates/static_page/index.html.eex

```diff
@@ -1,22 +1,85 @@
+<div class="container mx-auto my-5 py-10 text-center">
+    <section class="container mx-auto">
+      <h1 class="text-4xl text-center mb-6">Welcome to <%= Oli.VendorProperties.product_short_name() %></h1>
+      <p class="text-lg text-center text-gray-600 dark:text-gray-400 mb-10">The open-source online education platform for better learning outcomes</p>
 
-    <p class="my-2 py-2">
-      <%= link to: Routes.live_path(OliWeb.Endpoint, OliWeb.Projects.ProjectsLive), class: "btn btn-outline-primary mx-2" do %>
-        Authoring Workspace <i class="fa fa-arrow-right"></i>
-      <% end %>
-    </p>
+      <div class="grid grid-cols-1 md:grid-cols-3 gap-6 mt-12">
+        <div class="rounded-lg p-6">
+          <h2 class="text-2xl font-bold mb-4">Intuitive Authoring Experience</h2>
+          <p>Create engaging course materials with ease using our web-based editor. Build
+          adaptive learning experiences using a drag-and-drop flowchart. No coding skills
+          required!</p>
+        </div>
+
+        <div class="rounded-lg p-6">
+          <h2 class="text-2xl font-bold mb-4">Data-Driven Improvement</h2>
+          <p>Track learners' progress and analyze data in real-time to provide feedback and improve learning outcomes.</p>
+        </div>
 
-  <% else %>
-    <h1>Welcome to <%= Oli.VendorProperties.product_short_name() %></h1>
-    <p>Create, deliver and continuously improve course materials.</p>
+        <div class="rounded-lg p-6">
+          <h2 class="text-2xl font-bold mb-4">Learning Engineering Approach</h2>
+          <p>Design effective learning experiences that are instrumented and can be continuously improved using our learning engineering approach.</p>
+        </div>
+      </div>
 
-    <p class="my-2 py-2">
-      <%= link to: Routes.authoring_pow_session_path(@conn, :new), class: "btn btn-outline-primary" do %>
-        Authoring Sign In <i class="fa fa-arrow-right"></i>
+      <%= if author_signed_in?(@conn) do %>
+        <div class="text-center mt-12">
+          <%= link to: Routes.live_path(OliWeb.Endpoint, OliWeb.Projects.ProjectsLive), class:
+            "btn border border-blue-500 text-blue-500 hover:bg-blue-600 hover:text-white font-semibold py-3 px-8 rounded-full"
+           do %>
+            Authoring Workspace <i class="fa fa-arrow-right"></i>
+          <% end %>
+        </div>
+      <% else %>
+        <div class="text-center mt-12">
+          <%= link to: Routes.authoring_pow_session_path(@conn, :new), class:
+            "btn bg-blue-500 hover:bg-blue-600 text-white hover:text-white font-semibold py-3 px-8 rounded-full" do %>
+            Authoring Sign In <i class="fa fa-arrow-right"></i>
+          <% end %>
+        </div>
       <% end %>
-    </p>
-  <% end %>
 
+    </section>
+
+</div>
+<div class="bg-gray-50 dark:bg-gray-800 p-12 mt-12">
+    <section class="container mx-auto">
+      OLI Torus brings together an intuitive authoring experience, data-driven improvement, and
```

Comment:
I don't think we want static references to the product here.  This needs to use the vendor properties to allow folks to replace the product name.

Reply:
Certainly. I will convert to draft for now